tests/components/zwave_js/test_update.py

```diff
@@ -253,8 +282,15 @@ async def test_update_entity_sleep(
     async_fire_time_changed(hass, dt_util.utcnow() + timedelta(minutes=5, days=1))
     await hass.async_block_till_done()
 
-    # Because node is asleep we shouldn't attempt to check for firmware updates
-    assert len(client.async_send_command.call_args_list) == 0
+    # Two nodes in total, the controller node and the zen_31 node.
+    # The zen_31 node is asleep,
+    # so we should only check for updates for the controller node.
```

Comment:
Side note: I don't see a reason we shouldn't check for firmware updates of sleeping nodes. I wanted to remove that after the dead nodes fix, but forgot. Will try to remember for when I'm back.